app/controllers/posts_controller.py

```diff
@@ -104,10 +104,8 @@ def handlePost(page_to_load, request):
     attachments_ = None
   #print(new_post)
   # make post in canvas
-  if('announcement' in page_to_load):
-    title = 'Announcement'
-  else:
-    title = canvas_user.name 
+  canvas_user = CANVAS.get_user(1)
+  title = canvas_user.name
```

Comment:
It said that `canvas_user` wasn't defined and this was how I saw other controllers getting it.